### PR TITLE
fix for wwsh

### PR DESCRIPTION
**Pull request by pstaabp**
This fixes issue #1511 in that recent perl versions (at least 30) would not run a script using `wwsh`. 

Note: this should now run in the following forms:
```
wwsh myCourse addAdmin      # if WEBWORK_HOME is in PATH
./wwsh myCourse addAdmin    # inside bin directory
perl wwsh myCourse addAdmin  # inside bin directory
perl bin/wwsh myCourse addAdmin # inside $WEBWORK_HOME
```

### CODE REVIEW

**Review by drgrice1 — CHANGES_REQUESTED** [review]
This won't work.  The only thing here that is okay is the change of the first line of the file as discussed.  Your change to line 71 enforces that the script must be in the bin directory.  That is wrong.  Read my comment https://github.com/openwebwork/webwork2/issues/1511#issuecomment-1016953500.  I mentioned the way that this needs to be done.

**Comment by pstaabp** [comment]
> Your change to line 71 enforces that the script must be in the bin directory.

My thought is that all the scripts that are run using `wwsh` are in that directory.  I guess thinking a bit more, a user can write their own script which shouldn't be in that directory.

**Comment by drgrice1** [comment]
Yes, a script could be anywhere.  That is my point.

It sort of feels like you are thinking of the argument passed to the script as a command rather than a file.  A file is something that should have a relative or absolute file.   The usage of a command with a script would be more like what you are thinking with the command `wwsh myCourse addadmin`.   However, for a file (in this case a script file), the path should be given to tell the `wwsh` script where that file is.  So that means it should be `wwsh myCourse /path/to/addadmin`.

So basically, what I am trying to say is that this whole idea of making it so that you don't need to give the path is inherently incorrect.  I understand the desire to "make things easier", but that is just misguided here.

**Comment by drdrew42** [comment]
I have a bunch of wwsh scripts that are *not* in the webwork2/bin directory. They're custom for my institution and are not intended to be shared or monitored by git.

**Comment by pstaabp** [comment]
It seems like I'll just update the first line, but maybe build in some documentation on how to use since it isn't exactly clear how to use the script.  Should I use the `Getopt::Long` package like we've been doing, or just some custom help stuff?

**Review by drdrew42 — APPROVED** [review]

**Review by drgrice1 — APPROVED** [review]
I also approve!

**Comment by drgrice1** [comment]
I am going to merge this.  It has two approvals, and isn't that extensive.
